charter-template.html

```diff
@@ -334,10 +334,10 @@ <h2 id="participation">
           To be successful, this <i class="todo">(Working|Interest)</i> Group is expected to have 6 or more active participants for its duration, including representatives from the key implementors of this specification, and active Editors and Test Leads for each specification. The Chairs, specification Editors, and Test Leads are expected to contribute half of a working day per week towards the <i class="todo">(Working|Interest)</i> Group. There is no minimum requirement for other Participants.
         </p>
         <p>
-          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>.
+          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>. The group also welcomes non-Members to contribute technical submissions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
```

Comment:
It may be appropriate here to briefly suggest that submissions for consideration should follow the same guidelines for "incubated to reasonable maturity" as outlined in https://www.w3.org/Guide/standards-track/#criteria ? Along the lines of:

```suggestion
          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>. The group also welcomes non-Members to contribute technical submissions for consideration (provided they are <a href="https://www.w3.org/Guide/standards-track/#criteria">incubated to reasonable maturity</a>) upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
```

Reply:
I would prefer to stay with @jyasskin wording but change s/submissions/contributions/. I believe we should encourage contributions to the ongoing technical specifications and not encourage submissions outside of those (those should go to incubation instead).

Reply:
"contribute technical contributions" is a little repetitive...
```suggestion
          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>. The group also welcomes non-Members to make technical contributions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
```
or
```suggestion
          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>. The group also welcomes non-Members to offer technical contributions upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
```

? (This was pre-existing wording, but the idea that a "submission" is a larger piece of work that ought to be incubated makes sense to me.)

Reply:
If the intention of "non-Members to make technical contributions" is limited to existing or ongoing work, as opposed to new input documents (still within scope of the charter), then that works. But I'm not sure if that actually jumps out from the text (which is why I've attempted to distinguish between ongoing work and the incubation path).

Is this closer to what's intended:

```suggestion
          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>. The group also welcomes non-Members to make technical contributions for ongoing work, provided they agree to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
```

Reply:
Of the three suggested re-wordings, this one makes the most sense to me. Merge?